core/examples/http_bench.js

```
@@ -121,7 +156,19 @@ async function serve(rid) {
 }
 
 async function main() {
-  Deno.core.setAsyncHandler(handleAsyncMsgFromRust);
+  Deno.core.setAsyncHandler(HttpOp.handleAsyncMsgFromRust);
+  // Initialize ops by getting their ids from Rust
+  // and assign id for each of our ops.
+  const opsMap = Deno.core.getOps();
+  for (const [name, opId] of Object.entries(opsMap)) {
+    const op = opRegistry.get(name);
+
+    if (!op) {
+      continue;
```

Throw instead of continue